Diagnostics/README.md

```diff
@@ -0,0 +1,15 @@
+## Mesos Windows agent building and testing
```

Comment:
This is the `Mesos` `README`. Please make a simple `README` for Diagnostics or simply remove it and we'll add later.

Reply:
Agree they shouldn't be included.
I am trying to find out why those those extra files were added in this PR.

Reply:
I will remove it